lib/ansiblelint/utils.py

```diff
@@ -749,7 +749,9 @@ def normpath(path):
     Currently it generates a relative path but in the future we may want to
     make this user configurable.
     """
-    return os.path.relpath(path)
+    # convertion to string in order to allow receiving non string objects as
+    # arguments which would have failed under python2
```

Comment:
I think it's even not py2, but py3.4 and lower that don't fully support fspath protocol across stdlib

Reply:
@webknjaz Yep, you are right.